scripts/packaging/extensionsfolderpreupm.ps1

```diff
@@ -0,0 +1,80 @@
+<#
+.SYNOPSIS
+    Prepares the MRTK\Extensions folder for UPM packaging.
+.DESCRIPTION
+    Prepares the MRTK\Extensions folder for UPM packaging.
+.PARAMETER PackageRoot
+    The root folder containing the examples package contents. If not specified, the current folder is presumed.
+#>
+param(
+    [string]$PackageRoot
+)
+
+if (-not $PackageRoot) {
+    throw "Unknown package root path. Please specify -PackageRoot when building."
+}
+
+# This hashtable contains mappings of the sample categories to the folder which contains
+# the code and assets.
+#
+# Note that capitalization below in the key itself is significant.
+#
+# These paths are PackageRoot relative.
+$exampleFolders = [ordered]@{
+    "HandPhysicsService" = "HandPhysicsService\Examples";
+}
+
+# Beginning of the upm packaging script main section
+# The overall structure of this script is:
+#
+# 1) Create the required Samples~ folder
+# 2) Recursively copy the contents of the example folders into Samples~
+# 3) Prepare the samples entry for the package.json file
+# 4) Replace the %samples% token with the prepared data
+# 5) Overwrite the package.json file
+
+# Ensure the required folder exists
+$samplesFolder = "$PackageRoot\Samples~"
+if (-not (Test-Path -Path $samplesFolder)) {
+    New-Item $samplesFolder -ItemType Directory | Out-Null
+}
+
+# Copy each example folder
+foreach ($entry in $exampleFolders.GetEnumerator()) {
+    $sampleGroupName = $entry.Name
+    $sampleFolder = $entry.Value
+    Write-Output "Copying $PackageRoot\$sampleGFolder to $samplesFolder\$sampleGroupName"
+    Copy-Item -Path "$PackageRoot\$sampleFolder" -Destination "$samplesFolder\$sampleGroupName" -Recurse -Force
+    Copy-Item -Path "$PackageRoot\$sampleFolder.meta"-Destination "$samplesFolder\$sampleGroupName.meta"
+}
+
+# Create the samples data for the package.json file
+$sampleCount = 0
+$samples = "`"samples`": ["
+foreach ($entry in $exampleFolders.GetEnumerator()) {
+    # Since we need to place appropriate separator characters between entries, we need to
+    # keep track of how many folders we have processed
+    $sampleCount++
+
+    $folderName = $entry.Name
+
+    $displayName = "$folderName"
+    $description = "MRTK Examples: $folderName"
+    $path = "Samples~/$folderName"

```

Comment:
We should be consistent with the usage of separators (i.e. / or \) within this file.

Reply:
will do!

Reply:
consistent in all three scripts now.